<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assembler_Window</class>
 <widget class="QDialog" name="Assembler_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIP Assembly</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/main.qrc">
    <normaloff>:/icon/snowflake.ico</normaloff>:/icon/snowflake.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="main_section">
     <item>
      <layout class="QHBoxLayout" name="button_box">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
